### PR TITLE
Add suggest REST endpoint

### PR DESCRIPTION
**Pull request by kovacstomi**

### CODE REVIEW

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/suggest/SuggestRestService.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	}
+	
+	@Operation(
+			summary = "Concept suggestion", 
+			description = "Returns an actual concept of the specified code system based on the source term.")
+	@ApiResponses({ 
+		@ApiResponse(responseCode = "200", description = "OK"),
+		@ApiResponse(responseCode = "400", description = "Bad Request") 
+	})
+	@PostMapping
```
Comment (cmark) [inline core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/suggest/SuggestRestService.java]: Specify `consumes` for both this and the bulk endpoint, please.

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptSuggestionRequest.java [hunk 2]
```diff
@@ -68,7 +70,7 @@
 	@Min(1)
 	private int minOccurrenceCount;
 	
-	private transient List<String> topTokens;
+	private transient List<String> topTokens = Collections.emptyList();
```
Comment (cmark) [inline core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptSuggestionRequest.java]: I think the `emptyList()` assignment is not needed,

**Review by cmark — CHANGES_REQUESTED** [review]

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptSuggestionRequest.java (hunk 2 above)
Comment (kovacstomi) [reply core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptSuggestionRequest.java]: You are right, it's not needed 🎉

**Review by cmark — APPROVED** [review]
LGTM!

**Pull request merged**